src/commands/report-failure.test.ts

```diff
@@ -0,0 +1,68 @@
+import * as path from 'path';
+import { Polly } from '@pollyjs/core';
+import NodeHttpAdapter from '@pollyjs/adapter-node-http';
+import FSPersister from '@pollyjs/persister-fs';
+import reportFailure from './report-failure';
+import setupHardRejection from 'hard-rejection';
+
+class SanitizingPersister extends FSPersister {
+  static get id() {
+    return 'sanitizing-fs';
+  }
+
+  get options() {
+    return {
+      recordingsDir: path.resolve(__dirname, '__recordings__'),
+    };
+  }
+
+  // ensure that the authorization token is not written to disk
+  saveRecording(recordingId: string, data: any) {
+    data.log.entries.forEach((entry: any) => {
+      entry.request.headers = entry.request.headers.filter((h: any) => h.name !== 'authorization');
+    });
+    // @ts-ignore
+    return super.saveRecording(recordingId, data);
+  }
+}
```

Comment:
It seems a bit unfortunate that we have to manually munge to "sanitize" the on disk output, but I think "it's fine" probably 🤔

Reply:
patches welcome i'm sure